src/state/feed-feedback.tsx

```diff
@@ -301,28 +336,52 @@ function sendOrAggregateInteractionsForStats(
   }
 }
 
-function flushToStatsig(stats: AggregatedStats | null) {
+function flushToStatsig(
+  stats: AggregatedStats | null,
+  feedDescriptor: string,
+  isDiscover: boolean,
+) {
   if (stats === null) {
     return
   }
 
   if (stats.clickthroughCount > 0) {
-    logEvent('discover:clickthrough', {
+    if (isDiscover) {
+      logger.metric(
+        'discover:clickthrough',
+        {count: stats.clickthroughCount},
+        {statsig: false},
+      )
+    }
```

Comment:
I think the backwards compat layer is the right move for now, let's us gradually move to the new agnostic metric